src/schema/v2/notifications/__tests__/index.test.ts

```diff
@@ -28,6 +28,7 @@ describe("notificationsConnection", () => {
     const query = gql`
       {
         notificationsConnection(first: 10, notificationTypes: [ARTWORK_ALERT]) {
+          totalCount
```

Comment:
Does this differ from the total count field already exposed?

```graphql
{
  notificationsConnection(first: 5) {
    counts {
      total
    }
  }
}
```

Reply:
Ahhh, I see the other form is already exposed (via the helper we're using) but not threading the right value. I suppose it's fine to fix here and let the client choose how to best access.

Reply:
> I see the other form is already exposed (via the helper we're using)

Yes, that's right